test/config/expose-export-topref-false/schema.json

```diff
@@ -83,7 +110,9 @@
         "privateInterface",
         "privateAlias",
         "mixedInterface",
-        "mixedAlias"
+        "mixedAlias",
+        "publicAnonymousTypeLiteral",
+        "privateAnonymousTypeLiteral"
```

Comment:
Am I misunderstanding this completely, or should `privateAnonymousTypeLiteral` not appear here?

Reply:
No, I believe it just means that PrivateAnonymousTypeLiteral is not exposed in the "definitions" section.
Line 115 is just the field of the object being required.  
Note also privateAlias appears in the same object, but not in the definitions.